packages/react/src/components/Slider/useSlider.ts

```diff
@@ -338,61 +352,67 @@ export const useSlider = (props: ISliderProps, ref: React.Ref<HTMLDivElement>) =
     ? getNativeProps<React.HTMLAttributes<HTMLDivElement>>(buttonProps, divProperties)
     : undefined;
 
-  const labelProps = {
+  const labelProps: ILabelProps = {
     className: classNames.titleLabel,
     children: label,
     disabled,
     htmlFor: ariaLabel ? undefined : id,
   };
 
-  const valueLabelProps = showValue && {
-    className: classNames.valueLabel,
-    children: valueFormat ? valueFormat(value!) : value,
-    disabled,
-  };
+  const valueLabelProps: IValueProps | undefined = showValue
+    ? {
+        className: classNames.valueLabel,
+        children: valueFormat ? valueFormat(value!) : value,
+        disabled,
+      }
+    : undefined;
 
-  const lowerValueLabelProps = ranged &&
-    showValue && {
-      className: classNames.valueLabel,
-      children: valueFormat ? valueFormat(lowerValue!) : lowerValue,
-      disabled,
-    };
+  const lowerValueLabelProps: IValueProps | undefined =
+    ranged && showValue
+      ? {
+          className: classNames.valueLabel,
+          children: valueFormat ? valueFormat(lowerValue!) : lowerValue,
+          disabled,
+        }
+      : undefined;
 
-  const zeroTickProps = originFromZero && {
-    className: classNames.zeroTick,
-    style: getPositionStyles(originPercentOfLine),
-  };
+  const zeroTickProps: IStyleWithClassNameProps | undefined = originFromZero
```

Comment:
This should probably be `React.HTMLAttributes<HTMLElement>`. Same for the rest I think, though you might have to change the generic if it causes mismatches.